### PR TITLE
Bug 1602878 Prometheus State Variable update

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1602878

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :maple_leaf:

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -717,12 +718,17 @@ Set the following role variables to install and configure Prometheus.
 
 |Variable |Description
 
+|`openshift_prometheus_state`
+|The default value is present, which results in the installation or update of
+Prometheus. To uninstall Prometheus, set to absent.
```
Comment (mburke5678) [inline install_config/cluster_metrics.adoc]: @bmcelvee Should _present_ be formatted? _The default value is `present`..._

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -764,34 +770,30 @@ memory per host in the inventory.
...
 ----
 $ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/openshift-prometheus/config.yml
 ----
 
+[NOTE]
+====
+Make sure you have nodes labeled with `node-role.kubernetes.io/infra=true`, it
+is the default value for `openshift_prometheus_node_selector`. If you want to
+use other node selectors, please see xref:#openshift-prometheus-additional-deploy[Deploy Using Node-Selector].
+====
```
Comment (mburke5678) [inline install_config/cluster_metrics.adoc]: s/it is/which is  ?

**Comment by mburke5678** [comment]
@bmcelvee One nit and one question. LGTM!

**Pull request merged**